File: C/pset6/mario2.py
from cs50 import get_int

def main():

    A=get_int("Enter a number : ")

    for i in range(A):
        for s in range(A - i):
            print(" ",end="")

        for j in range(i + 1):
            print("#",end="")

        print()


if __name__ == "__main__":
    main()

File: C/pset1/Mario/mario2.c
#include <stdio.h>

int main(){
    int A,i,j,s,k;
    printf("enter a number!!");
    scanf("%d",&A);
        for(i=1;i<=A;i++){
            k=A-i;
            for(s=0;s<k;s++){
                printf(" ");
            }
            for(j=1;j<=i;j++){
                printf("#");
            }
            printf("\n");
        }
}